Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM scratch
```

Comment:
distroless nonroot is usually a better choice than scratch:
https://github.com/GoogleContainerTools/distroless/blob/main/examples/go/Dockerfile

/task would make it easier to COPY than /usr/local/bin/task in an image which is completely empty... 😅